parser/internal/parservX/parser_test.go

```diff
@@ -113,7 +113,7 @@ func transformToLegacyAST(expr ast.Expression) ast.Expression {
 
 func TestErrors(t *testing.T) {
 	assertParseFailure(t, "(person", "expected ')' following parenthesized expression", 0, 8)
-	assertParseFailure(t, "fn(person", "expected ')' following function arguments", 2, 10)
+	assertParseFailure(t, "count(person", "expected ')' following function arguments", 5, 13)
```

Comment:
couldn't this have been fn since it's missing a closing )? 🤔

Reply:
the error is "missing namespace", because it thinks it's defining a custom GROQ function.